### PR TITLE
Simplify LFI page and add Nginx file paths

### PR DESCRIPTION
**Pull request by faynopi**
TLDR; this commit will remove the repeated code and achieve the same effect using helper functions. Also this commit will add new Nginx files and add some small notes in different sections.

New Changes:
* Renamed some of the variable names to what I think are more meaningful names. [1]
* Renamed some of sections to better names. [2]
* Added PHP Manual links for wrapper filters.
* Remove the margin from the elements and add it to the root div tag.
* Remove marginTop because i think "Devider" component is enough space for the each section.
* Added Nginx section in "Useful LFI files".
* Added 4 Nginx files to the "Useful LFI files" section.
* Add some explanation for PHP remote file inclusion.
* Added line breaks in long paragraphs.
* This commit will introduce two helper function (mapFileEntries and copyButton) to build the DOM.
* Added a small note that "expect://" PHP wrapper filter is not enabled by default.

What has been removed:
* Removed repeated code for creating buttons for copy and URL encode. This is now done by using copyButton helper function.
* Removed repeated code for mapping file's array. This is now done by using mapFileEntries helper function.

---

1. Renamed variables:
    * `phpWrapperLfi` -> `phpWrapperExpect`
2. Renamed Sections:
    * "PHP Wrapper php://file" -> "PHP php://expect Wrapper RCE"
    * "PHP Wrapper php://file" -> "PHP Remote File Inclusion"

### CODE REVIEW

**Comment by faynopi** [comment]
@LasCC @rb-x

Please review the following code. also I forgot, the base should be `dev` branch not the master.

**Comment by rb-x** [comment]
Thank you for the PR @faynopi, excellent work :ok_hand:
